Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -644,3 +648,124 @@ def test_get_demisto_version(mocker):
         'version': '5.0.0',
         'buildNumber': '50000'
     }
+
+
+def test_assign_params():
+    from CommonServerPython import assign_params
+    res = assign_params(a='1', b=True, c=None, d='')
+    assert res == {'a': '1', 'b': True}
+
+
+class TestBuildDBotEntry:
```

Comment:
add tests that you would expect the result to be false, or an exception to be raised,,,